src/components/Scorecard.vue

```diff
@@ -50,23 +50,21 @@
             :key="metric"
```

Comment:
<img width="915" alt="Screenshot 2022-04-18 at 10 36 55 AM" src="https://user-images.githubusercontent.com/13185699/163757689-508c7bf6-6c4e-448d-b77b-4a9c4f79e60f.png">
you should be switching to the layout where the buttons are placed side-by-side instead of vertically top-down at a screen size of around 640px or so to avoid this. Had you checked this?

Reply:
discussed